table/namespace_handler.go

```diff
@@ -56,11 +57,23 @@ func (h *tableNamespaceHandler) Get(w http.ResponseWriter, r *http.Request) {
 	h.rd.JSON(w, http.StatusOK, nsInfo)
 }
 
+func readJSONRespondError(r *render.Render, w http.ResponseWriter, body io.ReadCloser, data interface{}) (err error) {
```

Comment:
I see this function twice times, can we move it to `apiutil`?.

Reply:
I didn't want to start creating new imports. I would appreciate the advise of the maintainers here if that is what is desired. I also didn't know why this handler is under /table instead of /server and why it uses the `apiutil` package instead of `api`.

Reply:
I think the duplicate code here is ok. The package `table` was originally designed to be independent of the other modules of the project (like a plugin for pd-server), so it is not dependent on other modules of pd as much as possible.